File: 2st/Yeonwoo/K번째수.py
def solution(array, commands):
    answer = []
    temp = []
    num = []
    for i in range(len(commands)):
        temp.append(array[commands[i][0]-1:commands[i][1]])
        temp[i].sort()
        answer.append(temp[i][commands[i][2]-1])
    return answer

File: 4st/Eunseong/K번째수.java
import java.util.*;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        for (int i = 0; i < commands.length; i++) {
            int start = commands[i][0] - 1;
            int end = commands[i][1] - 1;
            int k = commands[i][2] - 1;
            int[] temp = new int[end - start + 1];
            System.arraycopy(array, start, temp, 0, temp.length);
            Arrays.sort(temp);
            answer[i] = temp[k];
        }
        return answer;
    }
}
